assets/html/scss/documenter/components/_admonition.scss

```diff
@@ -21,6 +21,9 @@ $admonition-border-color: () !default;
   @if not map-has-key($admonition-border-color, $name) {
     $admonition-border-color: map-merge($admonition-border-color, ($name: $color)) !global;
   }
+  @if not map-has-key($admonition-border-color, $name) {
+    $admonition-border-color: map-merge($admonition-border-color, ($name: $color)) !global;
+  }
```

Comment:
Also, this change is unnecessary I think? It looks like the three lines just above it?

Reply:
Maybe it was a bad rebase with #2499 then